pkg/deploy/join.go

```diff
@@ -23,7 +23,6 @@ type WithJoinOptions struct {
 	IpsecDebug                    bool
 	SubmarinerDebug               bool
 	OperatorDebug                 bool
-	LabelGateway                  bool
```

Comment:
A flag can't just be removed. It needs to be deprecated first and then removed in the next release.

Reply:
This struct is introduced by this PR so it doesn't matter.